cypress/support/fragments/circulation/circulationsMenu.js

```diff
@@ -0,0 +1,17 @@
+export default class CirculationMenu {
```

Comment:
do we need this file in current situation with 2 places with menu paths: topmenu and settingsmenu?